### ISSUE TITLE
BUG: client.Transcripts.GetSubtitles Error: http2: response body closed

### ISSUE THREAD

**Issue by jackspirou** [AssemblyAI/assemblyai-go-sdk#20]
**Describe the bug**
I don't believe the `client.Transcripts.GetSubtitles` function works.  It errors with `http2: response body closed`.

**To Reproduce**
Follow the example here: https://www.assemblyai.com/docs/speech-to-text/speech-recognition#export-srt-or-vtt-caption-files

**Expected behavior**
`client.Transcripts.GetSubtitles` should return `[]byte` instead of error with the HTTP `body` already closed. 

**Additional context**
I believe the following is causing the problem:
- [Here](https://github.com/AssemblyAI/assemblyai-go-sdk/blob/56d61733abb638fbbbd0ea34480bb5e931766495/transcript.go#L194-L196) the `GetSubtitles` function does a `defer resp.Body.Close()` and then errors on the next line at `return io.ReadAll(resp.Body)`. 
- I believe this happens because the HTTP `body` is already closed earlier in the execution flow [here](https://github.com/AssemblyAI/assemblyai-go-sdk/blob/main/assemblyai.go#L145) via the `Client`s private `do` method.
- The `do` method is called inside of `GetSubtitles` [here](https://github.com/AssemblyAI/assemblyai-go-sdk/blob/56d61733abb638fbbbd0ea34480bb5e931766495/transcript.go#L190)

**Workaround**
I've just written my own function:
```
// Function to get the subtitles data
func GetSubtitles(ctx context.Context, transcriptID, format, apiKey string) ([]byte, error) {
	// Create the request URL
	url := fmt.Sprintf("https://api.assemblyai.com/v2/transcript/%s/%s", transcriptID, format)

	// Create a new GET request
	req, err := http.NewRequestWithContext(ctx, "GET", url, nil)
	if err != nil {
		return nil, fmt.Errorf("failed to create request: %w", err)
	}

	// Set the Authorization header
	req.Header.Set("Authorization", apiKey)

	// Perform the request
	client := &http.Client{}
	resp, err := client.Do(req)
	if err != nil {
		return nil, fmt.Errorf("request failed: %w", err)
	}
	defer resp.Body.Close()

	// Check for a successful response
	if resp.StatusCode != http.StatusOK {
		return nil, fmt.Errorf("unexpected status code: %d", resp.StatusCode)
	}

	// Read the response body
	body, err := io.ReadAll(resp.Body)
	if err != nil {
		return nil, fmt.Errorf("failed to read response body: %w", err)
	}

	return body, nil
}
```

**Comment by marcusolsson** [comment]
Thank you for reporting this, and sorry for the inconvenience 🙏 

I've fixed this upstream, and aiming to get a patch release out later today.

**Comment by marcusolsson** [comment]
Fix is available in v1.5.1. Let me know if you're still experiencing issues.
